drivers/volume/volume.go

```diff
@@ -50,7 +50,7 @@ type Driver interface {
 
 	// ValidateCreateVolume validates whether a volume has been created properly.
 	// params are the custom volume options passed when creating the volume.
-	ValidateCreateVolume(name string, params map[string]string) error
+	ValidateCreateVolume(appVols map[string]map[string]string, vState int, appName string, appToken string) error
```

Comment:
too much information here, can we use a wrapper struct to reduce the number of parameters passed?

Reply:
Modified slightly, can you please review  again